### PR TITLE
Update PolishFlagPack to 1.2.2

### PR DESCRIPTION
**Pull request by Venthe**
Update PolishFlagPack metadata to 1.2.2,
as contents of the mod are pure binary files,
no other change is required

### CODE REVIEW

**Comment by Venthe** [comment]
If I'm correct, autochecks are failed because of your Jenkins problems, or I've misunderstood?

```
Fetching latest ckan.exe
280 [1] WARN CKAN.KSPManager (null) - 7f34b7dadea974cc5375c68adcf188057d88bd32 at /home/jenkins/workspace/PR builder - NetKAN metadata/NetKAN/dummy_ksp is not a vaild install
```

File: NetKAN/PolishFlagPack.netkan [hunk 1]
```diff
@@ -6,7 +6,7 @@
     "author"       : [ "Venthe" ],
     "abstract"     : "Small pack of Polish flags",
     "license"      : "CC-BY-SA-4.0",
-    "ksp_version"  : "1.1.3",
+    "ksp_version"  : "1.2.2",
```
Comment (politas) [inline NetKAN/PolishFlagPack.netkan]: If this is nothing but flags, you could change that to 

    "ksp_version" : "any",
    "comment"     : "flagmod",

And then you won't need to update it again.

Comment (Venthe) [inline NetKAN/PolishFlagPack.netkan]: Done

**Comment by politas** [comment]
That looks great! Merging with thanks!

**Pull request merged**